code/controllers/subsystem/statpanel.dm

```diff
@@ -22,13 +22,17 @@ SUBSYSTEM_DEF(statpanels)
 		num_fires++
 		var/datum/map_config/cached = SSmap_vote.next_map_config
 		global_data = list(
-			"Map: [SSmapping.current_map?.map_name || "Loading..."]",
-			cached ? "Next Map: [cached.map_name]" : null,
-			"Round ID: [GLOB.round_id ? GLOB.round_id : "NULL"]",
-			"Server Time: [time2text(world.timeofday, "YYYY-MM-DD hh:mm:ss", world.timezone)]",
-			"Round Time: [ROUND_TIME()]",
-			"Station Time: [station_time_timestamp()]",
-			"Time Dilation: [round(SStime_track.time_dilation_current,1)]% AVG:([round(SStime_track.time_dilation_avg_fast,1)]%, [round(SStime_track.time_dilation_avg,1)]%, [round(SStime_track.time_dilation_avg_slow,1)]%)"
+			isnull(SSmapping.current_map) ? list("Loading") : \
+				SSmapping.current_map?.feedback_link ? \
+				list("Map: [SSmapping.current_map.map_name]", " (Feedback)", "action=openLink&link=[SSmapping.current_map.feedback_link]") : \
+				list("Map: [SSmapping.current_map?.map_name]"),
+			SSmapping.current_map.mapping_url ? list("same_line", " | (View in Browser)", "action=openWebMap") : null,
+			cached ? list("Next Map: [cached.map_name]") : null,
+			list("Round ID: [GLOB.round_id ? GLOB.round_id : "NULL"]"),
+			list("Server Time: [time2text(world.timeofday, "YYYY-MM-DD hh:mm:ss", world.timezone)]"),
```

Comment:
it worries me that we are creating htis many lists all the time, why do they have to be nested?

Reply:
3 items in the list, first one is text, second is text with a link, third is the link the second one brings you to. I can make lists not needed for places that don't have highlighted areas but I mostly just made this from what the MC panel does already though there they're all clickable

Reply:
Ok i got it and it does work

testing as blob so you can see ``other_str`` too
![image](https://github.com/user-attachments/assets/7fc80914-f33d-4ec9-854d-139379433efe)